pages/governance/pipeline_templates.md

```diff
@@ -28,35 +28,35 @@ An administrator can test a pipeline template against a pipeline using the _New
 
 If a template exists for the organization, it can be selected from the _Pipeline template_ dropdown to create a new build using the step configuration from that template.
 
-## Assigning a pipeline template to a pipeline
+## Requiring pipeline templates
 
-To permanently override a pipeline's step configuration with a pipeline template, administrators must first require pipeline templates via settings (see next section).
+The power of pipeline templates comes from how much you require their use. Administrators can select from the following options, listed in increasing strictness:
 
-Once pipeline templates are required, there are three options for assigning a template to individual pipelines:
+1. **Do not require pipeline templates:** Pipeline steps remain editable for any user with permission to create or update a pipeline. Templates can be tested against pipelines but cannot be assigned to them.
+1. **Require a pipeline template on new pipelines:** A template must be selected when creating a new pipeline. The step configuration of existing pipelines will become read-only. Pipelines can be assigned a template individually, making a gradual migration to pipeline templates possible.
+1. **Requiring a pipeline template for everything:** Templates are mandatory on all new and existing pipelines. When choosing this setting, you will select a pipeline template to apply to any pipeline that does not already have a template assigned.
 
-1. Selecting the template on the step settings for that pipeline (_Pipeline_ > _Settings_ > _Steps_).
-1. Using the REST API to [update the pipeline](https://buildkite.com/docs/apis/rest-api/pipelines#update-a-pipeline) with the appropriate `pipeline_template_uuid`.
-1. Using the GraphQL API [`pipelineUpdate` mutation](https://buildkite.com/docs/apis/graphql/schemas/mutation/pipelineupdate) with the appropriate `pipelineTemplateId`.
+>🚧 Changing requirements
+> When updating the requirements, you can only update the setting to an option that is more strict. Take care to test your pipeline templates before enforcing them on all pipelines.
 
-The correct IDs to use with the APIs for a template can be found on the template page in the UI.
+To change your organization's requirements for pipeline templates:
 
-> Note that pipelines that have not been setup using YAML steps cannot be assigned a pipeline template via the UI.
-> These pipelines must be either [migrated to YAML steps first](https://buildkite.com/docs/tutorials/pipeline-upgrade), updated via the APIs, or bulk-assigned a template when selecting the _Require a pipeline template for everything_ setting.
+1. Navigate to your [organization's pipeline template settings](https://buildkite.com/organizations/-/pipeline-templates).
```

Comment:
I've just realised that "organization's pipeline template settings" is a confusing phrase for that page given that within there is a "settings" page for pipeline templates.

Perhaps we avoid referring to this (here and earlier in the document) as "settings", i.e:

```suggestion
1. Navigate to your [organization's pipeline templates](https://buildkite.com/organizations/-/pipeline-templates).
```